.travis.yml

```diff
@@ -1,13 +1,20 @@
 language: python
-python:
-    - "3.6"
-    - "3.7"
-    - "3.8"
-    - "3.9"
+dist: focal
+
+jobs:
+  include:
+    - python: 3.6
+      env: TOXENV=py36
+    - python: 3.7
+      env: TOXENV=py37
+    - python: 3.8
+      env: TOXENV=py38
+    - python: 3.9
+      env: TOXENV=py39
```

Comment:
does this uses the latest minor version for 3.9?

Reply:
That's a good point/question!

Reply:
For the specific case of  `3.9` is using the latest version which is now `3.9.1`.
For the other python versions:
```
3.6.9 (3.6.12 is the latest now)
3.7.7 (3.7.9 is the latest now)
3.8.3 (3.8.7 is the latest now)
```
As Travis updates the Python build images, aliases for each version will point to different exact versions or patch levels.